### DISCUSSION TITLE
Why hpool haven't 5 seconds limitation?

### DISCUSSION THREAD

**Discussion by benkywong** [Chia-Network/chia-blockchain#4899] [Support]
2021-05-14T11:44:40.341 harvester chia.harvester.harvester: WARNING  Looking up qualities on \\192.168.1.9\File\plot-k32-2021-05-10-12-20-1e3327aa8ff1d603f95df81a9c2fc3317fd6bc5cd28b16a4f993bc1385b69e15.plot took: 9.640558004379272. This should be below 5 seconds to minimize risk of losing rewards.

I got a lot of this error in log. It is already a NAS with i5-2500k, 16G RAM. I got even more "5sec" errors when I connected to a remote NAS. How hpool works if there such a limit? I must join a pool instead of solo if I can only mount a remote NAS which look up time > 5sec?

**Comment by r1235613** [comment]
I suggest you use crontab to create temp file and del it every 3 min on your farming dict.
It can avoid your HDD asleep.

**Comment by k3vst3r** [comment]
Disable HDD sleep in edit power plan for windows.

USB HDD ignore this setting though, but you can disable them sleeping too by going to device manager an right clicking properties tab under power management don't allow windows to put this device to sleep option for HDD and any USB hubs also.

**Comment by benkywong** [comment]
The NAS is always busy due to every second has a plot copying to it. The remote NAS even. 

The point I am asking is how it works for HPool if my HDD look up time for file is already 4sec, the network delay 1sec is normal right? All uses on HPool directly attach HDD to computer instead of using NAS or even remote NAS?

**Reply by cccat6** [reply]
I guess HPool just didn't check it. Or maybe below 30 seconds will be fine for HPool. Actually, above 5 seconds will be risky but not a big NO. The big NO will be 30 seconds.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
